admin/commands/management/remove.go

```diff
@@ -55,7 +95,7 @@ func (cmd *removeMySQLCommand) Run() (commands.Result, error) {
 	}
 	_, err := client.Default.Service.RemoveService(params)
 	if err != nil {
-		return nil, err
+		return nil, errors.Errorf("We can't find service associated with the local node, %s", err)
```

Comment:
there might be errors not related to finding service(e.g. connection issues, or some internal server error), so this error message will be not relevant. We can check if cmd.ServiceID and cmd.ServiceName are not empty and based on that return an error in lines 85-86.